src/types.ts

```diff
@@ -196,6 +196,7 @@ export interface ExcalidrawProps {
   ) => JSX.Element;
   UIOptions?: UIOptions;
   detectScroll?: boolean;
+  detectPositionChange?: boolean;
```

Comment:
lastly, this should prob be named `detectPosition` to align with `detectScroll`?